src/libs/actions/FormActions.js

```diff
@@ -21,19 +21,10 @@ function setErrors(formID, errors) {
  * @param {Object} draftValues
  */
 function setDraftValues(formID, draftValues) {
-    Onyx.merge(`${formID}DraftValues`, draftValues);
-}
-
-/**
- * @param {String} formID
- * @returns {String}
- */
-function getDraftValuesKey(formID) {
-    return `${formID}DraftValues`;
+    Onyx.merge(`${formID}Draft`, draftValues);
```

Comment:
This is a function that is used for any form that has `shouldSaveDraft`, right? So will this change affect any other form?

Reply:
Ah yea, nice catch. Updated!